### ISSUE TITLE
urlSource.options is ignored

### ISSUE THREAD

**Issue by mariotaku** [ipfs/js-ipfs-utils#23]
https://github.com/ipfs/js-ipfs-utils/blob/e582304334584c2c076ee87a85063f6e0dd7c740/src/files/url-source.js#L9

Should this `options` to be passed to `ky.get(url, options)`?

**Comment by achingbrain** [comment]
According to the API docs, `urlSource` does not support any options - see 'Importing a file from a URL' here: [SPEC/FILES.md](https://github.com/ipfs/interface-js-ipfs-core/blob/f1077f710275ed6b222ecfa4357e956ddffc3ca8/SPEC/FILES.md#add) so the second argument to that function should probably be removed.

Is there something specific you're trying to do?

**Comment by mariotaku** [comment]
@achingbrain Yes. For example, if a url needs credentials to be downloaded, passing headers/cookies directly without having to make requests manually.

**Comment by achingbrain** [comment]
Sounds fair enough - would you like to submit a PR with a test or two that fixes the bug?

**Comment by mariotaku** [comment]
@achingbrain Sure! I'd love to do that, and I'll learn to write test for urlSource.
